Build fails with Kakadu 7.5

System: Ubuntu Linux 14.04, Kakadu 7.5

The build fails at the linking stage:

```
link: g++ -std=gnu++0x -g -O2 -o iipsrv.fcgi IIPImage.o TPTImage.o JPEGCompressor.o TileManager.o IIPResponse.o View.o Transforms.o Task.o OBJ.o FIF.o JTL.o TIL.o ICC.o CVT.o Zoomify.o DeepZoom.o SPECTRA.o PFL.o IIIF.o Watermark.o Main.o KakaduImage.o -fopenmp /home/ahankins/src/kdu/apps/make/jpx.o /home/ahankins/src/kdu/apps/make/jp2.o /home/ahankins/src/kdu/apps/make/kdu_stripe_decompressor.o /home/ahankins/src/kdu/apps/make/libkdu_v75R.so  -lmemcached -lz -lpthread ../fcgi/libfcgi/libfcgi.a -ljpeg /usr/lib/x86_64-linux-gnu/libtiff.so -lm -fopenmp
/home/ahankins/src/kdu/apps/make/kdu_stripe_decompressor.o: In function `kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int)':
kdu_stripe_decompressor.cpp:(.text+0x50a): undefined reference to `kd_supp_simd::ssse3_int16_to_uint8_rs_ilv3(unsigned char*, short**, int, int, int, bool, bool, int)'
kdu_stripe_decompressor.cpp:(.text+0x524): undefined reference to `kd_supp_simd::avx2_int16_to_uint8_rs_ilv3(unsigned char*, short**, int, int, int, bool, bool, int)'
.... (more of the same) ...
kdu_stripe_decompressor.cpp:(.text+0x9c4): undefined reference to `kd_supp_simd::ssse3_floats_to_int16_ilv1(short*, float**, int, int, int, bool, bool, int)'
collect2: error: ld returned 1 exit status
make[1]: *** [iipsrv.fcgi] Error 1
make[1]: Leaving directory `/home/ahankins/src/iipsrv/src'
make: *** [all-recursive] Error 1
```

Tested on Mac OSX as well, with the same error.

```
Undefined symbols for architecture x86_64:
  "kd_supp_simd::avx2_int16_to_int16_ilv1(short*, short**, int, int, int, bool, bool, int)", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
  "kd_supp_simd::avx2_floats_to_int16_ilv1(short*, float**, int, int, int, bool, bool, int)", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
  "kd_supp_simd::avx2_floats_to_uint8_ilv1(unsigned char*, float**, int, int, int, bool, bool, int)", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
... (more of the same) ...
 "kd_supp_simd::ssse3_int32_to_int16_rs_ilv1(short*, int**, int, int, int, bool, bool, int)", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
  "kd_supp_simd::avx2_stripe_transfer_static_init()", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
  "kd_supp_simd::ssse3_stripe_transfer_static_init()", referenced from:
      kd_supp_local::kdsd_tile::init(kd_supp_local::kdsd_component_state*, int) in kdu_stripe_decompressor.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [iipsrv.fcgi] Error 1
make: *** [all-recursive] Error 1
```

--- a/configure.in
+++ b/configure.in
@@ -227,7 +227,7 @@ if test "x${KAKADU}" = xtrue; then
 	LIBS="$LIBS $kakadu_path/apps/make/jpx.o $kakadu_path/apps/make/jp2.o $kakadu_path/apps/make/kdu_stripe_decompressor.o $kakadu_path/apps/make/ssse3_stripe_transfer.o $kakadu_path/apps/make/*.so -lpthread"
 
     AC_CHECK_FILE( $kakadu_path/apps/make/avx2_stripe_transfer.o,
-        AC_MSG_RESULT([configure: Kakadu is > v7.5.]);
+        AC_MSG_RESULT([configure: Kakadu is >= v7.5.]);
         KDU_75=true,
         AC_MSG_RESULT([configure: Kakadu is < v7.5])
     )

--- a/configure.in
+++ b/configure.in
@@ -224,7 +224,17 @@ if test "x${KAKADU}" = xtrue; then
 	AC_DEFINE(HAVE_KAKADU)
 	INCLUDES="$INCLUDES -I$kakadu_path/managed/all_includes/"
 	EXTRAS="KakaduImage.o"
-	LIBS="$LIBS $kakadu_path/apps/make/jpx.o $kakadu_path/apps/make/jp2.o $kakadu_path/apps/make/kdu_stripe_decompressor.o $kakadu_path/apps/make/*.so -lpthread"
+	LIBS="$LIBS $kakadu_path/apps/make/jpx.o $kakadu_path/apps/make/jp2.o $kakadu_path/apps/make/kdu_stripe_decompressor.o $kakadu_path/apps/make/ssse3_stripe_transfer.o $kakadu_path/apps/make/*.so -lpthread"
+
+    AC_CHECK_FILE( $kakadu_path/apps/make/avx2_stripe_transfer.o,
+        AC_MSG_RESULT([configure: Kakadu is > v7.5.]);
+        KDU_75=true,
+        AC_MSG_RESULT([configure: Kakadu is < v7.5])
+    )
+
+    if test "x${KDU_75}" = xtrue; then
+        LIBS="$LIBS $kakadu_path/apps/make/avx2_stripe_transfer.o $kakadu_path/apps/make/kdu_client_window.o"
+    fi
 fi
 
 INCLUDES="$INCLUDES -I."

